region/brinstar/kraid/Kraid Eye Door Room.json

```diff
@@ -651,6 +745,40 @@
       "bypassesDoorShell": true,
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharging, Gain Flash Suit (Spikesuit) - Leave with Shinespark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canFlashSuitDoubleSpark",
+        {"shineChargeFrames": 175},
+        "canDownBack",
+        "canShinechargeMovementTricky",
+        {"thornHits": 1},
+        "canSpikeSuit",
+        {"shinespark": {"frames": 19}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {
+          "position": "top"
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["ammo"], "requires": ["never"]}
+      ],
+      "note": [
+        "Requires precice movement. Down-back during the fall to clear the lower platform while shooting out the blocks."
+      ],
+      "devNote": [
+        "May be possible to open a green/red door with a super missile but it is on the edge of possibility.",
+        "Unmorphing on the first possible frame leaves in top position; second frame leaves in middle position."
```

Comment:
- move this line into `note`, like the others
- The frame you unmorph on effecting the height you leave at should be added to the tech description.  Only, this matters on the `canFlashSuitDoubleSpark` instead of the spikesuit, but not on every `canFlashSuitDoubleSpark` source.  I'd add something to the `canFlashSuitDoubleSpark` description.